esp/src/src/nls/es/hpcc.ts

```diff
@@ -331,15 +347,19 @@ export = {
     HideSpills: "Esconder derrames",
     High: "Alto",
     History: "Historia",
+    Homepage: "Página principal",
     HPCCSystems: "HPCC Systems®",
     Icon: "Icono",
     ID: "ID",
+    IgnoreGlobalStoreOutEdges: "Ignorar los bordes de salida de la tienda global",
```

Comment:
this is a very specialized message, but I think we need 
"almacén" instead of "tienda"